Tests/MongoSwiftSyncTests/UnifiedTestRunner/UnifiedCursorAndChangeStreamOperations.swift

```diff
@@ -3,6 +3,10 @@ import MongoSwiftSync
 import TestsCommon
 
 struct CreateChangeStream: UnifiedOperationProtocol {
+    /// Enables users to specify an arbitrary BSON type to help trace the operation through
```

Comment:
since we don't actually use this property I'd suggest we just add "comment" to `knownArguments` (with a TODO to remove it as part of SWIFT-1385), but don't bother to store it here or add it to `CodingKeys`. I think that should allow be enough to avoid errors while decoding the file.

once we support it, it will be an option on `ChangeStreamOptions` and stored on that type rather than directly on the operation here.

Reply:
Removed `comment` from everything except `knownArguments` and added `TODO`